components/authz-service/storage/v2/role.go

```diff
@@ -67,22 +55,3 @@ func NewUpdateRole(id string, name string, actions []string, projects []string)
 
 	return role, nil
 }
-
-func validateRoleInputs(id string, name string, actions []string, projects []string) error {
-	if emptyOrWhitespaceOnlyRE.MatchString(id) {
-		return errors.New(
-			"a role id is required and must contain at least one non-whitespace character")
-	}
-	if emptyOrWhitespaceOnlyRE.MatchString(name) {
-		return errors.New(
-			"a role name is required and must contain at least one non-whitespace character")
-	}
-	if len(actions) == 0 {
-		return errors.New("a role must contain at least one action")
```

Comment:
I see this check is covered by the proto file so OK to remove it here. (Just wondering if we are happy with that error message vs. this one, thinking about our move away from proto-validation. 🤔 ).

Reply:
🤔